package.json

```diff
@@ -20,7 +20,7 @@
     "test:e2e:safari": "mocha --safari",
     "test:server:start": "http-server &",
     "release": "np",
-    "release:prepare": "npm run release:prepare:build && npm run release:prepare:imports && npm run release:prepare:css && npm run release:prepare:commit",
+    "release:prepare": "npm run release:prepare:build && npm run release:prepare:imports && npm run release:prepare:css",
```

Comment:
@SanderKleykens ongewenste wijziging, ik pas het aan.

Reply:
Ja ik doe tegenwoordig graag ne release:prepare voor ik commit, maar ben de commit stap vergeten er terug in te steken.